contrib/storage-phoenix/pom.xml

```diff
@@ -326,6 +330,12 @@
           <argLine>-Xms2048m -Xmx2048m</argLine>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
```

Comment:
Could you please clarify what is the reason for adding this plugin?

Reply:
Without it I cannot build storage-phoenix using `-Phadoop-2`:

```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.275 s (Wall Clock)
[INFO] Finished at: 2022-02-09T19:44:28+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project drill-storage-phoenix: Could not resolve dependencies for project org.apache.drill.contrib:drill-storage-phoenix:jar:1.20.0-SNAPSHOT: Failure to find org.apache.directory.jdbm:apacheds-jdbm1:bundle:2.0.0-M2 in https://conjars.org/repo was cached in the local repository, resolution will not be reattempted until the update interval of conjars has elapsed or updates are forced -> [Help 1]
```

Reply:
Ok, thanks for the explanation. Yes, looks like `MiniKdc` depends on this library so it cannot be excluded. Could you please add this plugin under the hadoop-2 profile?